packages/kit/src/runtime/client/client.js

```diff
@@ -449,12 +449,12 @@ export function create_client(app, target) {
 
 			/** @type {import('@sveltejs/kit').LoadEvent} */
 			const load_input = {
-				route: {
-					get id() {
+				route: new Proxy(route, {
+					get: (target, key) => {
 						uses.route = true;
```

Comment:
This should only run if the key is id. 

Or do we wanna avoid that check under the assumption that anything else is user error anyway?

Reply:
My thought was to do it identically to the proxy on the server-side: 
https://github.com/sveltejs/kit/blob/32afba695088b946aefe96da75b36de9b0667fbe/packages/kit/src/runtime/server/page/load_data.js#L111-L124

If you'd like me to, I can change it though to check for the key.